docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -90,4 +97,135 @@ hs.refreshIndex("empIndex", "incremental")
 TODO
 
 ### Hybrid Scan
-TODO
+
+Hybrid Scan enables to utilize existing index data along with newly appended source files or
+deleted existing files, without explicit refresh operation. For appended files, Hybrid Scan works
+by changing the query plan to perform on-the-fly shuffle of newly appended data and merge with
+the index data. For deleted files, Hyperspace also modify the plan to exclude the rows from deleted files
+in the index data and for this, you need to set the lineage column config at index creation.
+
+This feature is disabled by default.
```

Comment:
"Currently, HybridScan is disabled by default. You can check the configuration page to see how it can be enabled."